src/domain/users/index.types.d.ts

```diff
@@ -39,14 +66,22 @@ type User = {
   readonly defaultAccountId: AccountId
   readonly deviceTokens: DeviceToken[]
   readonly createdAt: Date
-  phone: PhoneNumber
+  readonly phone: PhoneNumber
+  readonly twilio: PhoneMetadata | null
```

Comment:
same here, if you do the parsing in the service you can use here `phoneMetadata` as property name

Reply:
fixed